package.json

```diff
@@ -30,9 +30,9 @@
     "start": "npm run dev -s",
     "dev": "vite --config vite.demo.config.ts",
     "build": "vite build",
-    "build:demo": "vite build --config vite.demo.config.ts",
+    "build:demo": "node options_tester.cjs && vite build --config vite.demo.config.ts",
```

Comment:
should this not also go on "build"?

Reply:
No, build is for the npm package. The npm packages has no options file, therefore it would not make much sense. However, this options_tester should be a of the template